### PR TITLE
Implement rendering inside the servo UI

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by philn** [comment]
There's a small merge conflict. Can you please resolve it? I'll review the patches tomorrow most likely, I have some time off 😃

**Comment by sdroege** [comment]
I won't have time to update it today probably, but later this week should be possible :)

File: components/script/dom/htmlmediaelement.rs [hunk 1]
```diff
@@ -324,6 +446,10 @@ impl HTMLMediaElement {
             playground::player::PlayerEvent::EndOfStream => {
                 println!("EOS!");
             }
+            playground::player::PlayerEvent::FrameUpdated => {
+                println!("frame updated!");
```
Comment (philn) [inline components/script/dom/htmlmediaelement.rs]: This can be removed :)

Comment (sdroege) [reply components/script/dom/htmlmediaelement.rs]: Done, thanks :)

**Pull request merged**